### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+from typing import Any, Hashable
+
+
+@dataclasses.dataclass
 class Dictionary:
-    pass
+    capacity: int = 8
+    load_factor: float = 0.75
+    hash_table_: list = None
```
Comment (Bohdan-Salamakha) [inline app/main.py:10]: use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]
+
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+
+movies.clear()
+print(len(movies))
+
+movies.__setitem__(2, "Memento")
+print(len(movies))
+
+print(movies.__getitem__(2))
+movies.__setitem__(3, "Tenet")
+print(movies.__getitem__(3))
```
Comment (EdAlekseiev) [inline app/main.py:102]: Please, delete your test code

**Review by EdAlekseiev — APPROVED** [review]
Good job

File: app/main.py (hunk 2 above)
Comment (mwellick) [reply app/main.py:102]: sorry i forgot)

**Pull request closed without merging**